docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+services:
+  nginx:
+    image: "nginx:alpine"
+    ports:
```

Comment:
The only port we need to expose is the HTTPS port on 443 (unless we also direct requests to HTTP to HTTPS, but I don't think we need that on this):

```docker
ports:
  - 443:443
```